src/elements/button/button-static/button-static.visual.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffStandardStates } from '../../core/testing/private.js';
+
+import './button-static.js';
+
+// We test only the differences to the sbb-button and sbb-button-link
+describe(`sbb-button-static`, () => {
+  describeViewports({ viewports: ['zero'] }, () => {
+    for (const forcedColors of [false, true]) {
+      describe(`forcedColors=${forcedColors}`, () => {
+        for (const state of visualDiffStandardStates) {
```

Comment:
static button don't need focus since it cant be focused

Reply:
my plan was to ensure it's not focused :D

Reply:
oh, so it's fine as it is :D and maybe i should recheck some old commits of mine, because i feel i avoided that specific case

Reply:
The more I think I tend to remove it. :D

Reply:
I removed it :D__